test/fail_compilation/fail19931.d

```diff
@@ -0,0 +1,15 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19931.d(10): Error: `struct S` may not define both a rvalue constructor and a copy constructor
```

Comment:
(not that needs to be addressed in they PR but) that should read "... define both a**n** rvalue constructor..."